src/couch_log/src/couch_log_writer_stderr.erl

```diff
@@ -27,7 +27,7 @@ init() ->
 terminate(_, _St) ->
     ok.
 
-write(#log_entry{type = Type} = Entry, St) ->
```

Comment:
Is it worth keeping the `#log_entry{} = Entry` in the head?

Reply:
I think we should.

Reply:
Fixed in 9e8bb01.

Reply:
Well.. 2a70f6d now.